### PR TITLE
fix(blocks): ui update issue when color picker is opened for the first time

### PR DESCRIPTION
**Pull request by fundon**
Closes [BS-1026](https://linear.app/affine-design/issue/BS-1026/优化-color-picker-第一次打开时-ui-问题) [BS-1016](https://linear.app/affine-design/issue/BS-1016/mindmap-和-connector-卡死，缩放失效)

* fix ui update issue when color picker is opened for the first time and lazy listen
* ~~remove `--affine-palette-transparent` and make setting up transparent logic easier~~
* enhance the fallback logic for both `getPropertyValue` and `getColorValue`

### CODE REVIEW

**Comment by fundon** [comment]
* **#7788** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7788?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/utils.ts [hunk 1]
```diff
@@ -27,6 +27,7 @@ export function drawGeneralShape(
   ctx: CanvasRenderingContext2D,
   shapeModel: ShapeElementModel,
   renderer: Renderer,
+  filled: boolean,
   fillColor: string,
```
Comment (L-Sun) [inline packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/utils.ts:31]: What is the purpose of this new parameter?

File: packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/utils.ts [hunk 2]
```diff
@@ -50,7 +51,7 @@ export function drawGeneralShape(
 
   ctx.lineWidth = shapeModel.strokeWidth;
   ctx.strokeStyle = strokeColor;
-  ctx.fillStyle = fillColor;
+  ctx.fillStyle = filled ? fillColor : 'transparent';
```
Comment (fundon) [inline packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/utils.ts:54]: @L-Sun This has been missing before.

https://github.com/toeverything/blocksuite/blob/799772d22e6ed289e490893da3cf61d015f25bc2/packages/blocks/src/surface-block/canvas-renderer/element-renderer/shape/rect.ts#L77

**Comment by fundon** [comment]
I will add some tests for `getColorValue`

**Review by L-Sun — APPROVED** [review]

**Pull request merged**